src/client/pages/StaffViewAuthPage/index.js

```diff
@@ -91,35 +89,7 @@ function StaffViewAuthPage(props) {
       .then((response) => response.json())
       .then((data) => {
         setUserData(data);
-        if (data.authToken) {
```

Comment:
I believe we still need to update line 78 to pull the staff admin login.

Reply:
oops, thanks!